Gruntfile.js

```diff
@@ -21,6 +23,17 @@ module.exports = function(grunt) {
 
   grunt.config.set('compress', require('./grunt/config/compress'));
 
+  function spawnGulp(args, opts, done) {
+    grunt.util.spawn({
+      cmd: 'gulp',
+      args: args,
+      opts: assign({stdio: 'inherit'}, opts),
+    }, function() {
+      // TODO maybe add handling of status codes
+      done();
```

Comment:
Status code handling plz. This failed silently on my machine due to the above. ^